gdk-sys/src/lib.rs

```diff
@@ -685,6 +685,7 @@ extern "C" {
     // Gdk Key Handling                                                  NOT OK
     //=========================================================================
     pub fn gdk_keyval_name                         (keyval:c_uint) -> *mut c_char;
+    pub fn gdk_keyval_to_unicode                   (keyval:c_uint) -> u32;
```

Comment:
@gkoz: No, take a look [here](https://developer.gnome.org/gdk3/stable/gdk3-Keyboard-Handling.html#gdk-keyval-to-unicode). `u32` is perfectly fine.